test/ft/di_errors.cpp

```diff
@@ -414,9 +414,9 @@ test bind_any_of_not_related = [] {
 #if defined(__cpp_variable_templates)
 test bind_any_of_not_related_v = [] {
   auto errors_ = errors("constraint not satisfied",
-#if defined(__MSVC__) && _MSC_VER >= 1916
+#if defined(__MSVC__) && _MSC_VER >= 1916 && _MSC_VER < 1920
```

Comment:
@krzysztof-jusiak we need to get out of this business. It is just wast of time to fix these ifdefs.

I thing we instead we can just have error as `vector<vector<string>>` and if any of rows is conformed we are good